platform/engineering/design-docs/2020-12-10-auto-deploy-cms-build.md

```diff
@@ -0,0 +1,216 @@
+| Links checker       | Runs checks                                                                                                                                                                                                                                                                                                                                                                                                                                          | Runs checks                                                                                                                                                    |
+
+The VA.gov website can be deployed through the partial (or content-only) deploy or the full deploy.
+
+The full deploy is a daily job that deploys `vets-website` to the production environment of VA.gov. It uses the latest commit of `vets-website` as of 2:00pm ET on that day and the latest content downloaded from the Drupal server. It can be manually invoked as necessary, independently of the daily schedule.
+
+The content-only deploy builds `vets-website` with the latest content downloaded from the Drupal server and deploys to the dev and staging environments of VA.gov. This job can be triggered in one of two ways:
+- The daily production deploy of `vets-website`, which will build with latest release of `vets-website`
+- The `va-cms-bot`, which will build with the latest commit on `vets-website` in response to a manual trigger on the part of the CMS team
+
+#### Pain points
+
+- Content writers want to quickly draft, publish, and deploy content.
+  - However, `vets-website` is only deployed to production once per day.
+  - The CMS team can run content-only deploys independently of the daily production deploy, but that is triggered manually.
+- The content-only deploy does not include automated accessibility checks.
+- In local development, engineers can encounter confusing build errors that are due to the mismatch in outdated locally cached content and updated Liquid templates.
+
+### High Level Design
+
+This new auto-deploy has a few main differences with the existing content-only deploy.
+- Uses the CMS export to build content
+- Runs on a schedule instead of being triggered by other jobs (such as the daily prod deploy)
+- Deploys to production instead of only dev and staging
+
+The schedule will run between 8am and 8pm ET. It will initially be set to run hourly as a conservative cadence. It will then be incrementally adjusted to a higher frequency as the build system allows.
+
+To avoid deploying when content hasn't changed since the previous deploy, the job will compare the export data (e.g., using and archiving the checksums of the tarballs).
+
+## Specifics
+
+### Detailed Design
```

Comment:
Can we add a section about how to manually trigger a content only deploy?  This will still need to occur from the Drupal.